Documentation/specs/corehost.md

```diff
@@ -78,6 +78,8 @@ During host start-up, the host identifies if a `.deps` file is present and loads
 
 A `.deps` file is **not** required to successfully launch an application, but without it, all the dependent assemblies must be located within the same folder as the application. Also, since servicing is performed on the basis of packages, an application without a `.deps` file cannot use the servicing index to override the location of assemblies.
 
+The host looks for the `.deps` file in the Application Base with the file name `[AssemblyName].deps`. The path to the deps file can be overriden by specifying `--corehost-depsfile:{path to deps file}` as the first parameter to the application.
```

Comment:
I would suggest that the argument is simply "--depsfile:{path to deps file}" since this artifact is not unique to corehost but can be consumed by other artifacts in the CLI toolchain.